packages/synapse-react-client/src/mocks/msw/handlers/entityHandlers.ts

```diff
@@ -30,6 +30,22 @@ import { SynapseApiResponse } from '../handlers'
 import { uniqueId } from 'lodash-es'
 import { mockProjectsEntityData } from '../../entity/mockProject'
 import { mockUploadDestinations } from '../../mock_upload_destination'
+import { normalizeSynPrefix } from '../../../utils/functions/EntityTypeUtils'
+import { MockEntityData } from '../../entity/MockEntityData'
+
+function getMatchingMockEntities(entityIds: string[]): MockEntityData[] {
+  const normalizedPassedIds = entityIds.map(normalizeSynPrefix)
+  return mockEntities.filter(entity =>
+    normalizedPassedIds.includes(normalizeSynPrefix(entity.id)),
+  )
+}
+function getMatchingMockEntity(entityId: string): MockEntityData | undefined {
+  const matches = getMatchingMockEntities([entityId])
+  if (matches.length == 0) {
+    return undefined
+  }
+  return matches[0]
+}
```

Comment:
Update mock-service-worker functionality to be able to look up mock entities without a `syn` prefix.

Reply:
What do you think about logging a warning if multiple entities with the same id are found? Just wondering if it could help us catch when mock entities are added with ids that have already been used in existing mock entities